src/MainWindow.cpp

```diff
@@ -0,0 +1,936 @@
+                createInvalidKeyBox();
+            }
+
+            else
+            {
+                // Save both the group label, key, and value
+                QString groupLabel = splitKeys.at(0);
+                QString keyLabel = splitKeys.at(1);
+                QString value = keyValues.at(0);
+
+                // If the group label is the funds information section
+                if(groupLabel == m_FUNDS_INFORMATION_GROUP_LABEL)
+                {
+                    // If their is no total amount of money available currently stored in the config file
+                    if(value.isEmpty())
+                    {
+                        // Default the amount available to zero
+                        m_totalAmountAvailable = 0.00;
+                    }
+
+                    // If an amount available was found
+                    else
+                    {
+                        // Update the amount available to the config file's contents
+                        m_totalAmountAvailable = value.toDouble();
+                    }
+                }
+
+                // If the group label is a bill
+                else
+                {
+                    // If the bill being checked has not been inserted into the bill map yet
+                    if(!m_billMap.contains(groupLabel))
+                    {
+                        // Create a new Bill object
+                        Bill readBill;
+
+                        // Set the Bill's name
+                        readBill.setName(groupLabel);
+
+                        // Since the amount due will be read before the due date and the funding status, set the amount due of the Bill
+                        readBill.setAmountDue(value.toDouble());
+
+                        // Insert the Bill into the map mapped to the name of the bill
+                        m_billMap[groupLabel] = readBill;
+                    }
+
+                    // If the bill being checked already exists in the bill map, then we're checking its due date
+                    else
+                    {
+                        // If the key label is the due date key
+                        if(keyLabel == m_BILL_DUE_DATE_KEY)
+                        {
+                            // Set the appropriate Bill's due date from the bill map by converting the date string to a date object
+                            m_billMap[groupLabel].setDueDate(convertDateStringToDate(value));
+                        }
+
+                        // Otherwise, the key label is the funding status
+                        else
+                        {
+                            // Set the appropriate Bill's funding status in the bill map by converting the funding status string to a boolean
+                            m_billMap[groupLabel].setFundedStatus(fundingStatusStringToBoolean(value));
+
+                            if(m_billMap[groupLabel].isFunded())
```

Comment:
Add a comment here explaining how funded bills are being appended to the funded bills list for tracking purposes.

Reply:
Comment added.